### ISSUE TITLE
DeformConv2d offset and mask backprop

### ISSUE THREAD

**Issue by Cypher30** [pytorch/vision#5989]
### 🚀 The feature

I notice that the current vision of DeformConv2d module doesn’t treat offset and mask as parameters, so it seems that the standard backward routine cannot update the offset.  Is there any possibility that you could add this feature in the future?

### Motivation, pitch

I’m trying to build DNNs with above module, so it is not that convenient if I need to rewrite the backward process by myself

**Comment by datumbox** [comment]
Thanks for the feature request. We can look into doing this work on the future. It's not something we can pick up immediately because of lack of bandwidth, so note that you might need to find your own workarounds until we pick this up.

**Comment by Cypher30** [comment]
Thanks a lot for your reply!

**Comment by Zhaoyi-Yan** [comment]
@Cypher30 Maybe this work https://arxiv.org/abs/2107.03665 can help.

**Comment by Cypher30** [comment]
> @Cypher30 Maybe this work https://arxiv.org/abs/2107.03665 can help.

Thanks a lot!

**Comment by RGring** [comment]
+1
